src/components/home/ExploreListing/index.jsx

```diff
@@ -20,11 +21,21 @@ import {
  * @return {JSX.Element}
  */
 function ExploreListing() {
+  const navigate = useNavigate();
+
+  const handleSearch = (searchKey) => {
+    navigate(`/listing/search?key=${searchKey}`);
```

Comment:
Please use this string to utils/appPaths.js

```suggestion
    navigate(`${AppPaths.listingPaths.search}?key=${searchKey}`);
```